maven/maven.bzl

```diff
@@ -213,6 +214,7 @@ def _generate_maven_repository_impl(ctx):
                         target = artifact.third_party_target_name,
                         deps = _deps_string(normalized_deps),
                         artifact_coordinates = artifact.original_spec,
+                        use_jetifier = "\n    use_jetifier = True," if ctx.attr.use_jetifier else ""
```

Comment:
should the formatting (new line) just be handled in line 93? This was a weird line to see

Reply:
No. Because then you end up with a lot of extra spaces in the output.  It's a weird line to see, because it's template string replacement code.